src/nanoarrow/ipc/decoder_test.cc

```diff
@@ -856,14 +856,14 @@ TEST_P(ArrowTypeParameterizedTestFixture, NanoarrowIpcNanoarrowArrayRoundtrip) {
     EXPECT_EQ(ArrowIpcDecoderSetSchema(decoder.get(), schema.get(), &error), NANOARROW_OK)
         << error.message;
     EXPECT_EQ(ArrowIpcDecoderDecodeHeader(decoder.get(),
-                                          {buffer->data, buffer->size_bytes}, &error),
+                                          {{buffer->data}, buffer->size_bytes}, &error),
```

Comment:
all of the changes to this file suppress the warning of:

`/home/willayd/clones/arrow-nanoarrow/builddir/../src/nanoarrow/ipc/decoder_test.cc:859:44: warning: suggest braces around initialization of subobject [clang-diagnostic-missing-braces]`